packages/vite-plugin-windicss/src/index.ts

```diff
@@ -74,9 +74,14 @@ function VitePluginWindicss(userOptions: UserOptions = {}): Plugin[] {
     apply: 'serve',
     enforce: 'post',
 
-    configureServer(server) {
+    async configureServer(server) {
       if (utils.configFilePath)
         server.watcher.add(utils.configFilePath)
+
+      // NOTE: Track changes to the files so that they are re-scanned as needed.
+      // Added files are only detected if the user explicitly enables globbing.
+      const supportsGlobs = server.config.server.watch?.disableGlobbing === false
+      server.watcher.add(supportsGlobs ? utils.globs : await utils.getFiles())
```

Comment:
Something to consider if the user enables globbing, then the `excludes` provided in the plugin config don't really apply (Vite will watch everything that matches the globs).

In most cases, this shouldn't be a problem, since the main purpose of `excludes` is to avoid detecting tokens as Windi CSS utilities, but I thought I should bring this up anyway for consideration.